### ISSUE TITLE
[Feature Request] Line Chart: Split by Group

### ISSUE THREAD

**Issue by ajdapretnar** [biolab/orange3-timeseries#19]
Implement 'Group by' option as found in Distributions. This allows the user to view line chart per each, say, class value individually on the same plot.
Very handy! 👌

**Comment by janezd** [comment]
How would you distinguish between groups? Different colors already represent different variables.

Perhaps allow splitting by group only when showing a single line, so the colors can be used to represent groups?

**Comment by ajdapretnar** [comment]
I agree, this doesn't work with plots containing multiple variables. Probably the only solution is to limit this option to a single variable.
